package.json

```diff
@@ -14,7 +14,7 @@
     "lib/*.js"
   ],
   "devDependencies": {
-    "tap": "^14.9.2"
+    "tap": "^14.10.2"
```

Comment:
Doesn't have to be in this PR but I'd like to add `"engines": {"node": ">=10"}` before the next release.